common/app/implicits/Requests.scala

```diff
@@ -82,7 +83,7 @@ trait Requests {
     lazy val isHeadlineText: Boolean =
       r.getQueryString("format").contains("email-headline") || r.path.endsWith(HEADLINE_SUFFIX)
 
-    lazy val isModified = isJson || isRss || isEmail || isHeadlineText
+    lazy val isModified = isJson || isRss || isEmail || isHeadlineText || isApps
```

Comment:
this is so that we don't try and redirect to the "canonical" version of apps pages!